package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/dpp",
-  "version": "0.17.0-dev.4",
+  "version": "0.17.0-dev.5",
```

Comment:
It's better to bump version in a separate PR otherwise conventional commit changelog tool will ignore this commit for changelog.

Reply:
it's back to 0.17.0-dev.4 now